FourSquare/Controllers/Background/BackgroundViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  BackgroundViewController.swift
+//  FourSquare
+//
+//  Created by Mylo Ho on 8/1/16.
+//  Copyright © 2016 Le Van Binh. All rights reserved.
+//
+
+import UIKit
+import LGSideMenuController
+
+class BackgroundViewController: LGSideMenuController {
+
+    var leftViewController = LeftSideMenuViewController()
+
+    var allMenuItems = [ItemMenu]()
+
+    var activeMenuItems = [ItemMenu]()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        self.leftViewSwipeGestureEnabled = false
+        for i in 0..<6 {
+            let item = ItemMenu(item: MenuItemsSlide(rawValue: i)!, active: false)
+            allMenuItems.append(item)
+        }
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(self.UpdateItem), name: "ChangeItem", object: nil)
```

Comment:
- `UpdateItem`: function name starts with lower case
- `ChangeItem`: please define it